region/norfair/west/Crocomire Escape.json

```
@@ -677,6 +677,26 @@
         "Use the platform at the top-right of the room to gain temporary blue, then chain it through the door."
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "G-Mode Setup - Get Hit By Geruta",
+      "requires": [
+        "h_heatProof",
+        {"or": [
+          "canInsaneJump",
+          {"enemyDamage": {"enemy": "Geruta", "type": "contact", "hits": 1}}
+        ]}
```

```suggestion
        "h_heatProof",
        "canBePatient",
        {"or": [
          "canInsaneJump",
          {"enemyDamage": {"enemy": "Geruta", "type": "contact", "hits": 2}}
        ]}
```

The bat is extremely finicky.  
I don't think damageless is reasonable, without a full setup.  But maybe you could learn the movement cues.
The dragon is a nuisance where you can take extra time to avoid fireballs.